tests/auxtel/test_correct_pointing.py

```diff
@@ -87,6 +88,18 @@ async def test_configure(self):
 
                 self.assert_config(default_values, config)
 
+    async def test_check_center(self):
+        self.remotes_needed = False
+        async with self.make_script():
+            self.script.find_offset = unittest.mock.AsyncMock(
+                return_value=(np.nan, np.nan)
+            )
```

Comment:
Consider mocking `self.script.atcs.offset_xy` then asserting that it is not awaited afterwards?

Basically do:

```
self.script.atcs.offset_xy = unites.mock.AsyncMock()
```

then call 

```
self.script.atcs.offset_xy.assert_not_awaited()
```
after the 

```
with self.assertRaises(RuntimeError):
    await self.script._center()
````